book/website/ethical-research/data-hazards/dh-case-study.md

```diff
@@ -0,0 +1,281 @@
+(er-datahazardscasestudy)=
+# Case Study: Data Ethics and Reproducibility Symposium and Data Hazards Workshop 
+We (the organisers: Ceilidh Welsh and Susana Roman Garcia) wanted to create an event that revolved around the Data Hazards framework. The event focused on showcasing how different people think about embedding ethics into the work that they do. 
+As PhD students, we both felt the need to connect our interest in creating more ethical and reproducible work with our project's scientific questions. 
+This passion for working more ethically brought us together, and we sought to see how other people are embedding ethics into their work. 
+As part of the Symposium, we hosted talks from PhD students who have been striving to make their work reproducible, how university lecturers might think of implementing Data Hazards into their material, and more (this is available in this {ref}`er-datahazardscasestudy-postevent-githubrepo`). 
+Creating and hosting an event is an inspiring and fulfilling experience. 
+We were able to connect with others, share knowledge and learn new skills throughout the process. 
+In this chapter, we would like to share with you our experiences, including some of the challenges we faced, and provide tips for people hosting events in the future. 
+
+(er-datahazardscasestudy-overview)=
+## Overview 
+The Data Ethics and Reproducibility (DER) Symposium was a one-day event held at the Alan Turing Institute on March 10th 2023. 
+In total, we had eighteen attendees in-person and eighteen attendees online, not including volunteers and hosts. 
+This symposium aimed to showcase implementations of reproducibility and ethics for research, with a focus on Data Hazards.
+The symposium structure included a variety of speakers, an interactive workshop and networking opportunities. 
+
+A central part of this symposium was the {ref}`Data Hazards<er-datahazards>` Workshop to provide a collaborative training opportunity for attendees.
+The workshop encourages attendees to explore, discuss and reflect on the ethical implications and wider societal impact of specific data-intensive projects. 
+It was an opportunity for attendees to learn how to use the Data Hazards framework and see how it applies to different research projects.
+This allowed attendees to appreciate that ethics is complex, situational and important to discuss in our own contexts.
+
+**This case study is an account of our first-hand experiences organising and hosting an event. It may be useful to you if you would like to host or organise your own accessible workshop in data ethics and reproducibility (whether a Data Hazards workshop or not).** 
```

Comment:
Maybe make this part stand out from the rest of the text by adding [an admonition](https://the-turing-way.netlify.app/community-handbook/style/style-more-styling#special-content-blocks):

```
```{note}
**This case study is an account of our first-hand experiences organising and hosting an event. It may be useful to you if you would like to host or organise your own accessible workshop in data ethics and reproducibility (whether a Data Hazards workshop or not).**
```
```